packaging/helm/trento-server/Chart.yaml

```diff
@@ -6,7 +6,7 @@ description: The trento server chart contains all the components necessary to ru
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates
-version: 0.3.1
+version: 0.3.2
```

Comment:
Remember to upgrade the version of the chart we have above in this file, as it is used to create the SLE chart

Reply:
Sorry I'm not sure I have understood, I think I have updated all the chart versions. What am I missing?

Reply:
https://github.com/trento-project/trento/blob/main/packaging/helm/trento-server/Chart.yaml#L1
https://github.com/trento-project/trento/blob/main/packaging/helm/trento-server/Chart.yaml#L2